### ISSUE TITLE
GNOME Music no longer detected

### ISSUE THREAD

**Issue by Forage** [InputUsername/rescrobbled#58]
It looks like rescrobbled stopped working after GNOME Music changed changed the MPRIS identity from "GnomeMusic" to "org.gnome.Music". No tracks are being submitted any more.

"systemctl --user status rescrobbled.service" only gives me:
```
jan 22 16:10:23 Khazam-dum systemd[3842]: Started An MPRIS scrobbler.
jan 22 16:10:24 Khazam-dum rescrobbled[64205]: Authenticated with ListenBrainz successfully!
jan 22 16:10:24 Khazam-dum rescrobbled[64205]: Looking for an active MPRIS player...
```
If I change the white list to "vlc" and play a track it works as expected.

Using version 0.5.0 on Ubuntu 21.10.

**Comment by Forage** [comment]
Removing the white list completely identifies GNOME Music as just "Music".
This makes "playerctl --list-all" no longer a reliable way to identify the identity.
Is it a bug or by design that "org.gnome.Music" is not accepted?

**Comment by InputUsername** [comment]
That's weird. What does playerctl report?

As far as I know playerctl doesn't actually show the MPRIS identity but the D-Bus bus name. There might be something wrong with the way rescrobbled checks a player's identity/bus name though.

**Comment by Forage** [comment]
"org.gnome.Music" is what playerctl gives me, that's why I'm saying it's not reliable anymore, with just "vlc" for VLC.

**Comment by InputUsername** [comment]
Right, my bad. It looks like rescrobbled is parsing the bus name wrong, so `org.gnome.Music` is not accepted when it should be. There is a fix on the `fix-bus-name` branch, would you be able to test that?

**Comment by Forage** [comment]
Luckily the rust docker container makes it too easy to build the package (though oddly enough the binary is 9.7 MB instead of 3.5).
With the change both "org.gnome.Music" and "Music" get recognised and allow submitting tracks.

**Comment by InputUsername** [comment]
> With the change both "org.gnome.Music" and "Music" get recognised and allow submitting tracks.

Thanks for verifying! I've released `v0.5.1` with the fix.

> the binary is 9.7 MB instead of 3.5

I manually strip the binaries before releasing, I'm guessing the rust container doesn't do that.
